corehq/apps/reminders/migrations/0002_app_id.py

```diff
@@ -20,6 +18,7 @@ def _populate_app_id(apps, schema_editor):
 
 
 def _add_field(doc):
+    app_id_by_form_unique_id = {}
```

Comment:
This will reset the cache for each doc.  I think you could make it work like
```python
def _populate_app_id(...):
    ...
    app_id_by_form_unique_id = {}
    def add_field(doc):
        return _add_field(doc, app_id_by_form_unique_id)
    iter_update(SurveyKeyword.get_db(), add_field, with_progress_bar(doc_ids))

def _add_field(doc, app_id_by_form_unique_id):
    ...
````
Or by rewriting to use `IterDB` if that seems more straightforward.